lib/note-content-editor.tsx

```diff
@@ -113,6 +113,7 @@ class NoteContentEditor extends Component<Props> {
     }, SPEED_DELAY);
     this.props.storeFocusEditor(this.focusEditor);
     this.props.storeHasFocus(this.hasFocus);
+    this.contentDiv = createRef();
```

Comment:
ooh ooh! this can go up above right next to `decorations: string[] = []`

this is part of the fun of the class syntax. we'll additionally get class-level typing by putting it there and since it's there always we won't have to convince TypeScript that it truly exists - it will know.

Reply:
Ooh nice, this made TS happy